### DISCUSSION TITLE
Easily enable logging (used by dispatching) when testing backends

### DISCUSSION THREAD

**Discussion by eriknw** [networkx/networkx#7603] [Ideas]
I would sometimes find it useful to enable logging when running networkx tests with a backend. Would it be worth adding a `pytest` option to `networkx/conftest.py` and/or an environment variable to enable logging as shown in the docs?

https://networkx.org/documentation/latest/reference/backends.html#introspection

**Comment by rlratzel** [comment]
+1 for a pytest option.
